i18n/en.json

```diff
@@ -3899,6 +3897,11 @@
   "navbar_dropdown.teamSettings": "Team Settings",
   "navbar_dropdown.tryTrialNow": "Try Enterprise for free now!",
   "navbar_dropdown.userGroups": "User Groups",
+  "navbar_dropdown.userGroups.modal.description": "Create unlimited user groups with one of our paid plans. Get the full experience of Enterprise when you start a free, 30 day trial.",
```

Comment:
Looks like a number of texts reference a 30 day trial. Could we have each use `{trialLength}` or something similar and pass [FREEMIUM_TO_ENTERPRISE_TRIAL_LENGTH_DAYS](https://github.com/mattermost/mattermost-webapp/blob/master/utils/cloud_utils.ts#L36) in as a value?